<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>553</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="xcol_label">
        <property name="text">
         <string>xcol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="xcol_spinBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="ycol_label">
        <property name="text">
         <string>ycol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="ycol_spinBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="valcol_label">
        <property name="text">
         <string>valcol</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="valcol_spinBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QLabel" name="profilemode_label">
        <property name="text">
         <string>profile mode y col (Northing)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="profile_ycol_spinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="profile_rot_label">
        <property name="text">
         <string>profile rot N-&gt;E</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="profile_rot_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="y_offset_label">
        <property name="text">
         <string>Y (N) offset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="y_offset_doubleSpinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="x_EW_scale_label">
        <property name="text">
         <string>x EW scale 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="x_EW_scale_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="yz_NS_scale_label">
        <property name="text">
         <string>yz NS scale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="yz_NS_scale_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="rescale_pushButton">
        <property name="text">
         <string>rescale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="one_by_one_checkBox">
        <property name="text">
         <string>1:1 scale</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="x_EW_min_label">
        <property name="text">
         <string>x EW min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="x_EW_min_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="x_EW_max_label">
        <property name="text">
         <string>x EX max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="x_EW_max_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="dXlabel">
        <property name="text">
         <string>dX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="delta_ew_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QCheckBox" name="zero_x_ax_startcheckBox">
        <property name="text">
         <string>Zero x ax start</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="yz_NS_min_label">
        <property name="text">
         <string>yz NS min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="yz_NS_min_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="yz_NS_max_label">
        <property name="text">
         <string>yz NS max</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="yz_NS_max_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="dY_label">
        <property name="text">
         <string>dY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="delta_ns_doubleSpinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="IncX">
        <property name="text">
         <string>IncX</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="IncX_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="IncY">
        <property name="text">
         <string>IncY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="IncY_doubleSpinBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_17">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="IncXsub">
        <property name="text">
         <string>IncXsub</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="IncXsub_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="IncYsub">
        <property name="text">
         <string>IncYsub</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="IncYsub_doubleSpinBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_18">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="ann_x_label">
        <property name="text">
         <string>Ann. X major/minor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ann_x_maj_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ann_x_min_doubleSpinBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_21">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="ann_y_label">
        <property name="text">
         <string>Ann. y major/minor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ann_y_maj_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="ann_y_min_doubleSpinBox"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="Title_label">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="Title_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="EW_x_label">
        <property name="text">
         <string>EW xlabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="EW_x_label_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="NS_yz_label">
        <property name="text">
         <string>NS yz label</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NS_yz_label_lineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="ScaleTitle_label">
        <property name="text">
         <string>ScaleTitle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ScaleTitle_lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="ScaleUnit_label">
        <property name="text">
         <string>ScaleUnit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ScaleUnit_lineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="scale_ticks_label">
        <property name="text">
         <string>ann. every</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="Scale_annot_doubleSpinBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_22">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="mul_x_inputby_label">
        <property name="text">
         <string>mul  x EW input by</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="mul_x_inputby_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="mul_yz_inputby_label">
        <property name="text">
         <string>mul yz NS input by</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="mul_yz_inputby_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="mul_vals_inputby_label">
        <property name="text">
         <string>mul vals by</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="mul_vals_inputby_doubleSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="advals_label">
        <property name="text">
         <string>add to vals</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="advals_doubleSpinBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QCheckBox" name="logx_EW_checkBox">
        <property name="text">
         <string>log x EW</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="log_y_NS_checkBox">
        <property name="text">
         <string>log y NS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="log_vals_checkBox">
        <property name="text">
         <string>log vals</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_16">
      <item>
       <widget class="QLabel" name="mul_profile_ycol_inputby_label">
        <property name="text">
         <string>mul y profile input by</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="mul_profile_ycol_inputby_doubleSpinBox"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_29">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="log_y_profile_checkBox">
        <property name="text">
         <string>log y profile</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_28">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <item>
       <widget class="QCheckBox" name="contourlines_checkBox">
        <property name="text">
         <string>contour lines</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="topline_checkBox">
        <property name="text">
         <string>top line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="clip_above_topline_checkBox">
        <property name="text">
         <string>clip above top line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="bottomline_checkBox">
        <property name="text">
         <string>bottom line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="clip_below_bottomline_checkBox">
        <property name="text">
         <string>clip below bottom line</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="make_shell_pb">
        <property name="text">
         <string>make shell</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exec_shell_pb">
        <property name="text">
         <string>exec shell</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="exec_ghostview_checkBox">
        <property name="text">
         <string>exec gv</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>694</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
